### PR TITLE
Added retries on Eventing FI tests

### PR DESCRIPTION
**Pull request by mfaizanse**
**Description**

Changes proposed in this pull request:

- Added retries for Eventing FI tests

**Related issue(s)**

- https://github.com/kyma-project/kyma/issues/17146

### CODE REVIEW

File: tests/fast-integration/eventing-test/utils.js [hunk 1]
```diff
@@ -41,9 +41,9 @@ const kymaVersion = process.env.KYMA_VERSION || '';
 const kymaStreamName = 'sap';
 const isSKR = process.env.KYMA_TYPE === 'SKR';
 const skrInstanceId = process.env.INSTANCE_ID || '';
-const testCompassFlow = process.env.TEST_COMPASS_FLOW || false;
-const isUpgradeJob = process.env.EVENTING_UPGRADE_JOB || false;
-const isUpgradeJob2ndReconcile = process.env.EVENTING_UPGRADE_2ND_RECONCILE_JOB || false;
+const testCompassFlow = process.env.TEST_COMPASS_FLOW === 'true';
```
Comment (friedrichwilken) [inline tests/fast-integration/eventing-test/utils.js:44]: Why did you change true from bool to string?

Comment (mfaizanse) [reply tests/fast-integration/eventing-test/utils.js:44]: The ENV is a string. What happening was the if the ENV is defined then the `testCompassFlow` is true, regardless of whatever is in the value if this ENV.

Comment (friedrichwilken) [reply tests/fast-integration/eventing-test/utils.js:44]: Okay, I see. Thank you!

**Review by friedrichwilken — APPROVED** [review]
